### ISSUE TITLE
Changed behaviour for AbbreviatedMonthGenitiveNames?

### ISSUE THREAD

**Issue by jiimaho** [dotnet/runtime#1954]
I'm a bit unsure whether this is the expected behaviour or not.

Previously our code [in this jsfiddle](https://dotnetfiddle.net/RGA7ry) have been working as expected. This was on .NET Core 2.2

Now when upgrading to .NET Core 3.1 I had to add AbbreviatedMonthGenitiveNames.

Please see [my question on SO](https://stackoverflow.com/questions/59784036/abbreviatedmonthnames-not-working-properly-in-net-core-3-1) for further details and how to reproduce.

Is this a changed behaviour in the platform? Thanks in advance!

**Comment by khellang** [comment]
Is this on Windows or *nix?

**Comment by khellang** [comment]
I think this might be an intended change and a duplicate of https://github.com/dotnet/coreclr/issues/26370. I think the actual issue that lead to this change is https://github.com/dotnet/corefx/issues/2804.

// @tarekgh

**Comment by jiimaho** [comment]
It's on MacOS @khellang . Sorry I forgot to mention that.

**Comment by tarekgh** [comment]
@jiimaho what @khellang mentioned is correct. We had a reported bug https://github.com/dotnet/corefx/issues/2804 in formatting genitive names (e.g. when using `d MMM` formats). We have fixed this bug to correctly use the genitive month names and not just the abbreviated month names. 

You can fix your code by adding the second line in the snippet: 

```C#
AbbreviatedMonthNames = new [] { "jan", "feb",  "mar" ,"apr", "maj", "jun", "jul", "aug", "sep", "okt", "nov", "dec", ""}
AbbreviatedMonthGenitiveNames = AbbreviatedMonthNames;
```

For more information, you may consult the doc https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.abbreviatedmonthgenitivenames?view=netframework-4.8#System_Globalization_DateTimeFormatInfo_AbbreviatedMonthGenitiveNames

I'll try to reply on SO too.

**Comment by jiimaho** [comment]
Thanks for your input @tarekgh @khellang . You're right that this is a duplicate of https://github.com/dotnet/coreclr/issues/26370
 
Yes I did update our code with what you suggested @tarekgh . (Actually I updated our code before creating this issue, but was still unsure what the expected behaviour was so therefore I created this issue. Seem all is under control though and working as expected, so that's good). Thanks again.
